Cargo.toml

```diff
@@ -11,20 +11,20 @@ readme = "README.md"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-cargo_metadata = "0.6.4"
-docopt = "0.8"
-env_logger = "0.3"
-html5ever = "0.22"
-log = "0.3"
+cargo_metadata = "0.9"
+docopt = "1"
+env_logger = "0.7"
+html5ever = "0.24"
```

Comment:
There was a performance hit in `0.24`, can you upgrade to 0.25 instead? https://github.com/servo/html5ever/issues/414#issuecomment-645031801